pulsar-broker/src/test/java/org/apache/pulsar/broker/systopic/PartitionedSystemTopicTest.java

```diff
@@ -260,4 +261,33 @@ private void testSetBacklogCausedCreatingProducerFailure() throws Exception {
             Assert.fail("failed to create producer");
         }
     }
+
+    @Test
+    private void testSystemTopicNotCheckExceed() throws Exception {
```

Comment:
We should also test the max producer limitation for the system topic.

Reply:
Ah, I used to set policies to mock the producer. I should create writers directly...
Fixed.